lib/tests/streamlit/elements/exception_test.py

```diff
@@ -150,24 +160,94 @@ def func_with_error():
             f"Stack does not have length {original_stack_len}: {proto.stack_trace}",
         )
 
-    def test_uncaught_app_exception(self):
-        err = None
-        try:
-            st.format("http://not_an_image.png", width=-1)
-        except Exception as e:
-            err = UncaughtAppException(e)
-        self.assertIsNotNone(err)
-
-        # Marshall it.
-        proto = ExceptionProto()
-        exception.marshall(proto, err)
-
-        for line in proto.stack_trace:
-            # assert message that could contain secret information in the stack trace
-            assert "module 'streamlit' has no attribute 'format'" not in line
-
-        assert proto.message == _GENERIC_UNCAUGHT_EXCEPTION_TEXT
-        assert proto.type == "AttributeError"
+    def test_uncaught_app_exception_default_setting_community_cloud(self):
```

Comment:
nit: I think I would leave out community cloud here and refer to `False` as the generic purpose. Perhaps that `false` is used by community cloud could be a side note in the docstring